mdoc-js/src/main/scala-2.12/mdoc/modifiers/JsModifier.scala

```diff
@@ -220,7 +233,8 @@ class JsModifier extends mdoc.PreModifier {
       }
     val run = gensym.fresh("run")
     inputs += input
-    val id = s"mdoc-js-$run"
+    val htmlId = s"mdoc-html-$run"
+    val jsId = s"mdoc_js_$run"
```

Comment:
top-level name in SJS now has to be a valid identifier (because it's now a `let`?)

Reply:
This has always been an implementation detail so I think it's OK to change the identifier name.